components/brave_ads/core/internal/conversions/resource/conversion_resource_info.cc

```diff
@@ -29,6 +30,21 @@ ConversionResourceInfo& ConversionResourceInfo::operator=(
 
 ConversionResourceInfo::~ConversionResourceInfo() = default;
 
+// static
+base::expected<ConversionResourceInfo, std::string>
```

Comment:
As discussed we should remove any unrelated changes for the ML flatbuffer changes. Thanks